src/components/SyncContainer.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import withRouter from 'react-router-dom/withRouter';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { locationChange } from '../actions/syncActionCreators';
+
+function mapDispatchToProps(dispatch) {
+	return {
+		dispatchLocationChange: bindActionCreators(locationChange, dispatch),
+	};
+}
+
+/**
+ * @module SyncContainer
+ */
+export class SyncContainer extends React.Component {
+	/**
+	 * This container only cares about routing changes. When the route changes,
```

Comment:
Okay, I'll reword. This container essentially serves the role of `react-router-redux`, which would dispatch an action each time the route changed. Since `react-router-redux` [doesn't support v4](https://github.com/reactjs/react-router-redux/issues/520#issuecomment-277528454), this component will directly dispatch the `LOCATION_CHANGE` action when the route changes.